web/src/manifest.json

```diff
@@ -1,13 +1,11 @@
 {
-    "name": "d-installer",
-    "requires": {
-        "cockpit": "137"
-    },
-
-    "tools": {
-        "index": {
-            "label": "D-Installer"
-        }
-    },
-    "content-security-policy": "connect-src 'self' https://localhost:9090 wss://localhost:9090 ws://localhost:8080"
+  "name": "d-installer",
+  "requires": {
+    "cockpit": "137"
+  },
+  "tools": {
+    "index": {
+      "label": "D-Installer"
+    }
+  }
 }
```

Comment:
Do we have any reason to change the indentation? I would keep it as other Cockpit modules.

Reply:
No, we don't have it. What is more, I didn't realize about it. Will fix it in a while

Reply:
Fixed